tests/build_/test_arrangements.py

```diff
@@ -1198,6 +1198,7 @@ class TestArrangementV5(TestArrangementV4):
                 'group_manifests',
                 'pulp_tag',
                 'pulp_sync',
+                'import_image',
```

Comment:
Do we need a test to ensure the plugin parameters were rendered correctly?

Reply:
I'll code it if you think we need it.